### PR TITLE
Add CscCore to CrossPlatform.sln

### PR DESCRIPTION
**Pull request by agocke**

### CODE REVIEW

**Comment by agocke** [comment]
@jaredpar @jasonmalinowski FYI

**Comment by agocke** [comment]
And I'm going to file a bug on MSBuild for this issue and create a comment in our code pointing to it.

**Comment by jaredpar** [comment]
@agocke which issue are you referring to?

File: build/Targets/VSL.Imports.targets [hunk 1]
```diff
@@ -49,6 +49,16 @@
     </When>
   </Choose>
 
+  <!-- Work around mono bug -->
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == 'DNXCore' and '$(OS)' != 'Windows_NT'">
```
Comment (RichiCoder1) [inline build/Targets/VSL.Imports.targets]: @jaredpar I assume @agocke is referring to this.

Comment (agocke) [inline build/Targets/VSL.Imports.targets]: Correct.

**Comment by jasonmalinowski** [comment]
:+1:

**Pull request merged**